Makefile

```diff
@@ -94,6 +95,20 @@ lint:
 	npm run lint ||  EXIT_STATUS=$$?; \
 	exit $${EXIT_STATUS}
 
+.PHONY: lint-quick
+lint-quick:
+	EXIT_STATUS=0; \
+	npm run lint-staged ||  EXIT_STATUS=$$?; \
+	$(VIRTUAL_ENV)/bin/python manage.py makemigrations --dry-run --check --noinput || EXIT_STATUS=$$?; \
+	exit $${EXIT_STATUS}
+
+.PHONY: lint-python-files
+lint-python-files:
+	EXIT_STATUS=0; \
+	$(VIRTUAL_ENV)/bin/isort --df -c $(ARGUMENTS) || EXIT_STATUS=$$?; \
```

Comment:
I think the ARGUMENTS are still missing? And yes, I think it makes sense to also keep the help section updated?! Or what do you think?

Reply:
I didn't add as the make check-migrations wasn't added so assumed only things that needed to be there are commands that might be run by someone, I figured this probably wouldn't be run by someone but will add it anyway. Also thinking about it the make check-migrations doesn't need to be there anymore as that is called in the make lint-quick so will rm.
As for the ARGUMENTS thing, I don't know, I think this is something @rmader added, i'll ask in morning as stuck in style linting in Opin now.

Reply:
The Arguments should just be a variable added to the beginning of the file.